chrome/content/zotero/zoteroPane.js

```diff
@@ -2786,6 +2786,16 @@ var ZoteroPane = new function()
 				return;
 			}
 			
+			//Special treatment for zotero links
+			if (uri.match(/(^zotero\:\/\/)((open-pdf)|(select))/)) {
+				var ioservice = Components.classes["@mozilla.org/network/io-service;1"]
+					  .getService(Components.interfaces.nsIIOService);
+				var zoteroLink = ioservice.newURI(uri, null, null);
+				var openZoteroLink = Components.classes["@mozilla.org/network/protocol;1?name=zotero"]
```

Comment:
This last line doesn't need to be assigned to anything, and the above assignments would be better as `let`, not `var`.